### DISCUSSION TITLE
creating github key

### DISCUSSION THREAD

**Discussion by shawn-trial** [community/community#122601] [Programming Help]
### Body

hello, i'm trying to add ssh key to ssh agent. When I try to open the ssh/config file in the terminal , all I see is empty lines with purple tilde at the start. Even when I paste the changes I want to make, I can't press enter. Don't know what to do

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ddaeducation** [comment]
import pandas as pd

**Reply by shawn-trial** [reply]
sorry I'm new to this. pandas? pd?

**Comment by brandonyee-cs** [comment]
Restart your computer to fix the issue with ssh/config files in the terminal. As for adding a ssh key, I would recommend using GitHub CLI instead and creating a GitHub Key in setting --> developer settings --> personal access tokens.

**Comment by jamesleeksc** [comment]
C:\Users\Administrator\.ssh\
Just create a config file in this path and add content.
But this is a possible method on WINDOWS.
